yt_dlp/extractor/tv5mondeplus.py

```diff
@@ -151,16 +148,19 @@ def process_video_files(v):
         duration = (int_or_none(try_get(metadata, lambda x: x['content']['duration']))
                     or parse_duration(self._html_search_meta('duration', webpage)))
 
-        description = self._html_search_regex(
-            r'(?s)<div[^>]+class=["\']episode-texte[^>]+>(.+?)</div>', webpage,
-            'description', fatal=False)
+        title = episode = self._html_search_regex(r'<h1 class="main-title">([^<]+)', webpage, 'title', default=None)
+        series = self._html_search_regex(r'<p class="video-title">([^<]+)', webpage, 'title', default=None)
+        subtitle = self._html_search_regex(r'<p class="video-subtitle">([^<]+)', webpage, 'subtitle', default=None)
+        if subtitle:
+            episode = subtitle
 
-        series = self._html_search_regex(
-            r'<p[^>]+class=["\']episode-emission[^>]+>([^<]+)', webpage,
-            'series', default=None)
+        ep_summary = self._search_regex(
+            r'<div[^>]+class="ep-summary"[^>]*>(.+?)</div>', webpage,
+            'episode summary', fatal=False, flags=re.DOTALL)
 
-        if series and series != title:
-            title = '%s - %s' % (series, title)
+        description = self._html_search_regex(
+            r'<p class="text">(.+?)</p>', ep_summary,
+            'description', fatal=False, flags=re.DOTALL)
```

Comment:
could we simplify this with `get_element_by_class` (utils)?

Reply:
Yes. Changed this for some other fields as well.